macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -62,7 +62,8 @@
     'ignore_small_changes': {
       'spike_failure_percent_threshold': none,
       'drop_failure_percent_threshold': none
-    }
+    },
+    'include_other_warehouse_specific_columns': false,
```

Comment:
why not just `include_warehouse_specific_columns`? why the `other`?

Reply:
Because for BigQuery users, I'll add BigQuery fields and this `var` does not affect it.
It does affect if I'm a BigQuery user and whether it'll add Redshift fields.